table-join/table-join.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python
+
+"""
+Join the columns of two tables based on given key columns, using
+(pandas dataframe.merge option). Assumes that the input file has a
+header line with column names.
+"""
+
+import argparse
+import sys
+import pandas as pd
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("-t","--table1",
+                    help = "Input first table file; reads from STDIN if '-'")
+parser.add_argument("-g","--table2",
+                    help = "Input second table file; reads from STDIN if '-'")
+parser.add_argument("-c","--key-table1", default= "transcript",
```

Comment:
Rename to `key1`.

Reply:
Currently it only supports 1 column as key. The perl version supports multiple. Pandas can also support multiple. So use nargs to make it a list. Pandas should be able to deal with it directly.

Reply:
Remove the default. This needs to be a required option.